support/mender-inventory-geo

```diff
@@ -34,6 +34,8 @@ ATTR_NAME_IP="geo-ip"
 ATTR_NAME_CITY="geo-city"
 ATTR_NAME_COUNTRY="geo-country"
 ATTR_NAME_TIMEZONE="geo-timezone"
+ATTR_NAME_LAT="geo-lat"
+ATTR_NAME_LON="geo-lon" # loc=latitude,longitude
```

Comment:
strange place for the _# loc=latitude,longitude_ comment

Reply:
fixed, I hope you like it.

Reply:
Edit: I know you like it :) because we discussed privately :)